frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/ResultsTable.js

```diff
@@ -155,7 +155,7 @@ const ResultsTable = ({ tableContainerClass, config,data,isLoading,isFetching,fu
                 globalSearch={config?.enableGlobalSearch ? filterValue : undefined}
                 onSearch={config?.enableGlobalSearch ? searchQuery : undefined}
                 data={searchResult}
-                totalRecords={data?.count || data?.TotalCount}
+                totalRecords={data?.count || data?.TotalCount || data?.totalCount}
```

Comment:
@jagankumar-egov Earlier I only checked inbox APIs there key was totalCount only hence replaced it
But in search API its coming as TotalCount, hence updated code to accept both